Week5/MyFavoriteMovies/app/src/main/java/com/raywenderlich/myfavoritemovies/model/MovieDatabase.kt

```diff
@@ -0,0 +1,10 @@
+package com.raywenderlich.myfavoritemovies.model
+
+import androidx.room.Database
+import androidx.room.RoomDatabase
+
+@Database(entities = [(Movie::class)], version = 1)
+abstract class MovieDatabase : RoomDatabase() {
```

Comment:
You could've placed the database building function here, instead of manually doing it in the App class